.travis.yml

```diff
@@ -46,6 +46,16 @@ matrix:
     - php: 7
       env:
         - DEPS=latest
+    - php: 7.1
+      env:
+        - DEPS=lowest
+    - php: 7.1
+      env:
+        - DEPS=locked
+        - CHECK_CS=true
```

Comment:
Probably we don't need to run CS check on PHP7.1, usually we run it only on one version, and as it is on PHP7 here is redundant.